### ISSUE TITLE
CI on master is broken because of schemars again

### ISSUE THREAD

**Issue by apoelstra** [rust-bitcoin/rust-bitcoin#3393]
cc #3200 most recent instance before this (40 days ago).

Let's just delete schemars.

**Comment by tcharding** [comment]
In all fairness the last problem was my fault not the fault of `schemars`. Do you mean delete `extendend_tests/schemars/` or delete the code out of `hashes`? If the later, I can do the patch but I don't know how to justify it, are you able to suggest a commit message justification please.

**Comment by tcharding** [comment]
I removed the test crate in #3395, can this issue now track removal of the implementations?

**Comment by apoelstra** [comment]
The commit message can be "we introduced this as a personal favor to Jeremy Rubin, and it broke our CI over and over and over but eventually it seemed like it was stable (mainly, our MSRV caught up with its MSRV) so we just let it slide, and in the meantime Jeremy isn't even working on bitcoin stuff anymore, plus he didn't get much value in having `schemars` on `hashes` but nowhere else in the rust-bitcoin ecosystem".

Or something.

Basically, we don't want to be supporting schemars forever unless it has zero cost. And if it's breaking CI it's not zero cost.

**Comment by yancyribbens** [comment]
> Jeremy isn't even working on bitcoin stuff anymore,

That's too bad :(.   Hopefully this doesn't _actually_ go in a commit message though.  Seems too informal, although Hopefully he returns to Bitcoin in the future.
